### ISSUE TITLE
Modify for object detection

### ISSUE THREAD

**Issue by nikky4D** [mlfoundations/wise-ft#4]
Is it possible to use this with a multilabel dataset? with up to 5 objects per image?

**Comment by mitchellnw** [comment]
Yes this will probably still work, though we don't have code currently for this.

**Comment by vkrishnamurthy11** [comment]
I tried using this for a multi label dataset where each input can belong to one or more of two categories. I get the following error:
`RuntimeError: stack expects each tensor to be equal size, but got [1] at entry 0 and [2] at entry 4`

I think some code change will be required if we want to support this functionality.
